Library/Formula/tmux.rb

```diff
@@ -43,14 +43,26 @@ def install
 
     system "make", "install"
 
-    bash_completion.install "examples/bash_completion_tmux.sh" => "tmux"
-    pkgshare.install "examples"
+    if head?
```

Comment:
I'd be fine with that. I don't think the extra folder click for stable users is going to hurt anyone.